AlphaWallet/Tokens/Types/TokenObject.swift

```diff
@@ -28,7 +28,7 @@ class TokenObject: Object {
     @objc dynamic var _info: TokenInfoObject?
 
     var info: TokenInfoObject {
-        get { return _info! } //NOTE: Better to make it crash than create dummy object
+        get { return getTokenInfo() } //NOTE: Better to make it crash than create dummy object
```

Comment:
Should we remove the comment?

Reply:
Yes, I will do it. Also found the reason of it, TolenUpdate, I missed to instantiate info there. 🤦‍♂️